Mathlib/Algebra/BigOperators/Balance.lean

```diff
@@ -0,0 +1,59 @@
+/-
+Copyright (c) 2024 Yaël Dillies, Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Bhavik Mehta
+-/
+import Mathlib.Algebra.BigOperators.Expect
+
+/-!
+# Balancing a function
+
+This file defines the balancing of a function `f`, namely the unique function `g` such that
```

Comment:
If you define the function like this in the first sentence, you should at least prove that `balance` is this unique function! Imo we should just say what the definition is here, and match the docstring below.

Reply:
Like so?